apps/src/lib/script-editor/LevelTokenDetails.jsx

```diff
@@ -202,16 +228,34 @@ export class UnconnectedLevelTokenDetails extends Component {
             {this.props.level.ids.length > 1 && (
               <div>
                 <hr style={styles.divider} />
-                <span style={styles.levelFieldLabel}>Active</span>
-                <input
-                  type="radio"
-                  onChange={this.handleActiveVariantChanged.bind(this, id)}
-                  defaultChecked={id === this.props.level.activeId}
-                  style={styles.checkbox}
-                  name={`radio-${this.props.stagePosition}-${
-                    this.props.level.position
-                  }`}
-                />
+                <span>
+                  <span style={styles.levelFieldLabel}>Active</span>
+                  <input
+                    type="radio"
+                    onChange={this.handleActiveVariantChanged.bind(this, id)}
+                    defaultChecked={id === this.props.level.activeId}
+                    style={styles.checkbox}
+                    name={`radio-${this.props.stagePosition}-${
+                      this.props.level.position
+                    }`}
+                  />
+                </span>
+                {id !== this.props.level.activeId && (
+                  <span style={styles.removeVariant}>
+                    <button
+                      onMouseDown={this.handleRemoveVariant.bind(this, id)}
```

Comment:
we don't need to `.bind` the method here since we're using the fat-arrow syntax on `handleRemoveVariant()` ([interesting article about this](https://medium.com/@nikolalsvk/loosing-bind-this-in-react-8637ebf372cf)). instead, this could look like:

```
<button
  onMouseDown={event => this.handleRemoveVariant(id)}
/>
```

or, since we aren't using the `event` arg in our event handler:

```
<button
  onMouseDown={() => this.handleRemoveVariant(id)}
/>
```